components/application-connectivity-validator/internal/validationproxy/handler.go

```diff
@@ -82,7 +82,7 @@ func NewProxyHandler(
 		appRegistryHost:          appRegistryHost,
 
 		eventsProxy:      createReverseProxy(eventServiceHost),
-		eventMeshProxy:   createReverseProxy(eventMeshHost, withRewriteBaseURL("/"), withEnforceURLHost),
+		eventMeshProxy:   createReverseProxy(eventMeshHost, withRewriteBaseURL("/")),
```

Comment:
imho it would be more obvious to use this request option everywhere instead of removing it from the mesh proxy and secretly sneak it in to the reverseproxy as a default

Reply:
Changed for all other sneaky changes as well.